target/linux/ath79/image/generic.mk

```diff
@@ -438,6 +438,13 @@ define Device/jjplus_ja76pf2
 endef
 TARGET_DEVICES += jjplus_ja76pf2
 
+define Device/kuwfi_cpe2g-9531
+  ATH_SOC := qca9531
+  DEVICE_TITLE := KUWFI CPE2G-9531
+  IMAGE_SIZE := 8000k
```

Comment:
The firmware partition is 0x780000, so this would be 7680k. Is there some reason why you have 8000k or is this an error?

Reply:
fixed